app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt

```diff
@@ -404,52 +344,12 @@ class LoginActivity : AppCompatActivity() {
             "join_date" to Date(),
             "picture" to user.photoUrl.toString()
         )
```

Comment:
Same as before, you could use the real userProfile class, it already get converted to a nice firebase format, and there is some methods to upload to the DB